src/styles/variables.scss

```diff
@@ -1,6 +1,38 @@
+// Color branding below according to Publicis Sapient colors
+
+// Primary coolor is used as the baseline for the theme to appear in important
```

Comment:
Not very confident that it will be updated in the long term 🤔

Reply:
What do you mean?

Reply:
👀